.github/workflows/linux.yml

```diff
@@ -119,6 +119,8 @@ jobs:
         run: |
           make dist
       - name: Update version
+        if: |
+          github.ref == 'refs/heads/master'
```

Comment:
Renaming maintenance branch name rule to `maintenance/11.03` from `v11.03-maintenance` and changing this to `startsWith(github.ref, 'refs/tags/') || startsWith(github.ref, 'refs/heads/maintenance/')` may be better.

Reply:
"Update version" execute only on master.
So, I think that the following code is correct. Is that my misunderstanding?

```
if: |
  !startsWith(github.ref, 'refs/tags/') && !startsWith(github.ref, 'refs/heads/maintenance/')
```

Reply:
Right.
My code was wrong.